geonode/maps/admin.py

```diff
@@ -50,6 +50,14 @@ class MapAdmin(TabbedTranslationAdmin):
     form = MapAdminForm
     actions = [metadata_batch_edit]
 
+    def delete_queryset(self, request, queryset):
+        """
+        We need to invoke the 'ResourceBase.delete' method even when deleting
+        through the admin batch action
+        """
+        for obj in queryset:
+            obj.delete()
```

Comment:
@afabiani isn't [queryset,delete()](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#delete) better here (and the other cases)?

Reply:
@giohappy this is intentional, if we are not going to call `delete()` on each single object, the bulk delete from the admin dashboard will completely bypass the overridden `delete` methods and signals... Django misteries

Reply:
I see... Not nice, but I undersatnd.